bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -3044,12 +3035,12 @@ TEST_F(BcStTest, bkpTestRvbDataConflictDetection) {
       ASSERT_TRUE(!memcmp(defaultRvbDataDigest.content(), rvbDataDigest.content(), sizeof(Digest)));
     } else {
       DigestGenerator digestCtx;
-      Digest rvbDataDigest;
+      Digest clearRvbDataDigest;
```

Comment:
sorry, I probably miss something, why the name change?

Reply:
rvbDataDigest hides other variable of same name in the outer scope.